synapse/cortex.py

```diff
@@ -6179,6 +6181,533 @@ async def iterTagPropRows(self, layriden, tag, prop, form=None, stortype=None, s
         async for item in layr.iterTagPropRows(tag, prop, form=form, stortype=stortype, startvalu=startvalu):
             yield item
 
+    async def _initVaults(self):
+        self.vaultsdb = self.slab.initdb('vaults')
+
+        self.vaulttypes = {}
```

Comment:
All lookup structures should be tables within the slab that use dupsort to reference values which are vault IDs to lookup via the main vaultsdb table.